### ISSUE TITLE
Deleted logs come back after logoff-login

### ISSUE THREAD

**Issue by horsus** [telegramdesktop/tdesktop#1524]
I am unable to delete message history.

I've tried clearing histories and deleting conversations in various combinations — as soon as I quit and log in again, history is back. I've tried manually deleting the app (including local files) and reinstalling. 

Problem is present in both group chats and individual conversations, but not all of them — I've been able to delete chats with certain users, but not all. I'm not sure, but seems that problems occur with older larger chats with a lot of shared media/links etc. Several of my friends have the same problem.

I have no other sessions open.

Deleting individual messages works fine.

Telegram version is 0.9.18

OS is Elementary OS 0.3.2 Freya 64 bit

**Comment by defcon** [comment]
Same probleme here.

OS is Archlinux 64bit

**Comment by Zenexer** [comment]
Is it still broken after running this?

``` bash
sudo chown -R $USER:$USER ~
```

**Comment by defcon** [comment]
Standard in Arch or manjaro is $USER:$GROUP and i don't want to change my homedir completely :)
Nonetheless the ~ is complete in $USER:$GROUP style incl.  ~/.TelegramDesktop ...

**Comment by telegramdesktop** [comment]
This should be fixed for some versions now, please check 0.9.32 (current stable).

**Comment by Protonator** [comment]
I recently tried to delete the chat logs of all my Telegram Desktop (V2.3) contacts on both (!) sides. It worked flawlessly with the exception of two of my oldest contacts. The chat logs with them reach way back to 2016. When I delete the logs on both sides (no matter if I use "Delete chat" or "Clear history"), it seems to work on first sight, but after I restart the client, the logs are back again.

I examined this further and saw that "Delete chat" or "Clear history" sometimes deletes only around one month from the past and sometimes nothing. Then I have to manually delete 100 messages and use "Delete chat" or "Clear history" again until the whole chat history is finally gone. This is an exhausting pile of click work:

– Use "Delete chat" or "Clear history" on both sides
– Restart client
– Check if deletion worked or not by switching to the beginning of the chat calendar
– If not, delete 100 messages from the beginning manually
– Then use "Delete chat" or "Clear history" on both sides again
– Restart client
– …

It would be nice if this long known bug could finally getting fixed on client or server side. If I could help with any tests or program logs, feel free to ask.
